crates/ruff_workspace/src/resolver.rs

```diff
@@ -305,13 +307,15 @@ pub fn python_files_in_path(
         }
     }
 
+    // Open questions: How does force_exclude work with lint options
+    // How to avoid filtering out files that have been explicitly past.
```

Comment:
```suggestion
    // How to avoid filtering out files that have been explicitly passed.
```

Good question, intuitively i'd say force exclude applies to a subcommand overall so it applies to all excludes